src/view/element.js

```diff
@@ -123,6 +123,17 @@ export default class Element extends Node {
 	}
 
 	/**
+	 * Length of this view element in model if it is mapped. This value can be overwritten in classes extending
+	 * {@link engine.view.Element Element} to provide correct position mapping in {@link engine.conversion.Mapper Mapper}.
+	 *
+	 * @readonly
+	 * @type {Number}
+	 */
+	get modelLength() {
```

Comment:
Yep, `Mapper.registerViewElementLength( ViewElementClass, lengthCallback );` is fine.